Gemfile.lock

```diff
@@ -570,9 +568,9 @@ GEM
 PLATFORMS
   arm64-darwin-21
   ruby
+  x86_64-darwin-19
```

Comment:
Happy to trash this gemfile lock change, but figured I would flag that my recent dev setup triggered these changes

Reply:
Yes you should drop this.